src/styles/blocks/product.scss

```diff
@@ -0,0 +1,90 @@
+.product {
+  display: flex;
+  flex-direction: column;
+
+  padding: 32px 16px 16px;
+  width: 200px;
+  box-sizing: border-box;
+
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  &__pic-box {
+    display: flex;
+    margin: 0 3px 40px;
```

Comment:
maybe better to manage img size with `height` and `width`? not with `margin`.

Reply:
here you can leave only `margin-bottom: 40px;` and set appropriate img size below in `&__pic`